addons/account/static/src/js/tax_group.js

```diff
@@ -0,0 +1,253 @@
+odoo.define('account.tax_group', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var fieldRegistry = require('web.field_registry');
+    var AbstractField = require('web.AbstractField');
+    var fieldUtils = require('web.field_utils');
+    var QWeb = core.qweb;
+
+    var TaxGroupCustomField = AbstractField.extend({
+        resetOnAnyFieldChange: true,
+        events: {
+            'click .oe_tax_group_editable span': '_onClick',
+            'keydown .oe_tax_group_editable .oe_tax_group_amount_value input': '_onKeydown',
+            'blur .oe_tax_group_editable .oe_tax_group_amount_value input': '_onBlur',
+        },
+
+        /*
+            !! Important information !!
+            ---------------------------
+            FYI:
+            this.record.data.tax_group_amount:
+            [0] => Tax group name
+            [1] => Tax group amount
+            [2] => Base amount
+            [3] => Formated tax group amount
+            [4] => Formated base amount
+            [5] => Length (Useless ?)
+            [6] => Tax group id
+            [7] => Tax ids for a tax group
+            [8] => Tax group amount before change
+        */
+
+        //--------------------------------------------------------------------------
+        // Public
+        //--------------------------------------------------------------------------
+
+        /**
+         * This method always returns true to display the field.
+         * 
+         * @public
+         * @override
+         * @returns always true
+         */
+        isSet: function () {
+            return true;
+        },
+
+        //--------------------------------------------------------------------------
+        // Private
+        //--------------------------------------------------------------------------
+
+        /**
+         * This method is called by "_setTaxGroups". It is 
+         * responsible for calculating taxes based on
+         * tax groups and triggering an event to
+         * notify the ORM of a change.
+         * 
+         * @param {Array} taxGroup 
+         */
+        _changeTaxValueByTaxGroup: function (taxGroup) {
```

Comment:
this method only needs taxGroup[7] and the deltaAmount, it is called only from _setTaxGroup, therefore it might be better to only give those two argument to the method as it will be easier to read and not always think of the structure of taxGroup. Maybe rename the method also -> _changeTaxValue: function(tax_group_id, deltaAmount) and in the _setTaxGroup instead of changing the taxGroup, you could directly pass the correct information. You might even delete the 8 argument of your structure as it is no longer necessary.